src/views/pages/address/enter-address-form.njk

```diff
@@ -3,167 +3,84 @@
 {% set validateNameOrNumber = "validateInput(this, 50, '" + i18n.address.enterAddress.errorMessages.nameOrNumberMissing | escape + "', '" + i18n.address.enterAddress.errorMessages.nameOrNumberLength | escape + "');" %}
 {% set validateAddressLine1 = "validateInput(this, 50, '" + i18n.address.enterAddress.errorMessages.addressLine1Missing | escape + "', '" + i18n.address.enterAddress.errorMessages.addressLine1Length | escape + "');" %}
 {% set validateTownOrCity = "validateInput(this, 50, '" + i18n.address.enterAddress.errorMessages.townOrCityMissing | escape + "', '" + i18n.address.enterAddress.errorMessages.townOrCityLength | escape + "');" %}
-{% set validatePostcode = "validateInput(this, 15, '" + i18n.address.enterAddress.errorMessages.postcodeMissing | escape + "', '" + i18n.address.enterAddress.errorMessages.postcodeLength | escape + "');" %}
 
-<form class="form" action={{ props.currentUrl }} method="post">
-    <input type="hidden" name="pageType" value={{ props.pageType }}>
-    {% include "includes/csrf_token.njk" %}
+{% include "includes/csrf_token.njk" %}
```

Comment:
i'd remove the csrf tokens import out of here and have it where the <form is added on each page